AirCasting/Notes/NotesHandler.swift

```diff
@@ -6,9 +6,10 @@ import Foundation
 protocol NotesHandler {
     func addNote(noteText: String)
     func deleteNote(note: Note)
-    func updateNote(note: Note, newText: String)
+    func updateNote(note: Note, newText: String, completion: @escaping () -> Void)
     func getNotes(completion: @escaping ([Note]) -> Void)
     func fetchSpecifiedNote(number: Int, completion: @escaping (Note) -> Void)
+    func fetchSession(completion: @escaping (SessionEntity) -> Void) 
```

Comment:
fetching single session should not be present in this interface. But if you change the code as I suggested in the VM comment you should be able to not expose this one (use it internally in controller - this is fine ofc)